README.md

```diff
@@ -20,8 +24,50 @@ Below we list examples of issues reported in this way:
 |   [5312](https://github.com/verilator/verilator/issues/5312)  |  [Verilator](https://github.com/verilator/verilator) |     Crashes when calling a function created in "generate" block.        |
 |   [1174](https://github.com/steveicarus/iverilog/issues/1174) |  [Icarus Verilog](https://github.com/steveicarus/iverilog) | Crashes when assigning to parameters in a procedural block. | 
 |   [4598](https://github.com/YosysHQ/yosys/issues/4598)  |  [Yosys](https://github.com/YosysHQ/yosys) |     Crashes while simplifying program.|
-  
-If you ever use ChiBench to find bugs in some EDA tool, we would appreciate it very much if you could reach out to us and report your experience.
+
+### Chigen
+
+**ChiGen** is a  tool designed for synthesizing realistic Verilog designs to test and debug Electronic Design Automation (EDA) tools. Originally developed to validate the [Jasper Formal Verification Platform](https://www.cadence.com/en_US/home/tools/system-design-and-verification/formal-and-static-verification.html), ChiGen has proven to be highly effective in identifying bugs across a wide range of tools, including Verible, Verilator, and Yosys. A tutorial to get started with ChiGen can be found below. 
+
+
+
+
+## ChiGen Tutorial 
+### Installing
+#### 1. Clone Chimera: 
+  ```
+  git clone  https://github.com/lac-dcc/chimera.git
+  ``` 
+
+#### 2. Build Verible
+The verible binary is in the `verible_bin` folder. However, if the binary does not work on the computer, try building it from [source](https://github.com/chipsalliance/verible)
```

Comment:
Refer to our fork as the source: https://github.com/joaovam/verible/tree/73f291dc73d38e8f876ce9cc35aad62ff932f138.